README.md

```diff
@@ -9,12 +9,54 @@
 ![](https://github.com/astarte-platform/astarte-message-hub/actions/workflows/build.yaml/badge.svg?branch=master)
 [![codecov](https://codecov.io/gh/astarte-platform/astarte-message-hub/branch/master/graph/badge.svg)](https://app.codecov.io/gh/astarte-message-hub)
 
-A central service that runs on (Linux) devices for collecting and delivering messages from N apps using 1 MQTT connection to Astarte.
+A central service that runs on (Linux) devices for collecting and delivering
+messages from N apps using 1 MQTT connection to Astarte.
+
```

Comment:
Set line length not longer than 100 characters.

Reply:
Reformatted the file